### ISSUE TITLE
Carousel Blocks is not accessible for keyboard only users

### ISSUE THREAD

**Issue by grahamarmfield** [godaddy-wordpress/coblocks#2305]
### Describe the bug:

This is a significant accessibility issue for keyboard users.

I've added an XL Carousel to a page. I can choose the picture in the Carousel by clicking on the thumbnails beneath the large image. But there is no way for keyboard users to operate this Carousel as thumbnail images do not get keyboard focus.

### To reproduce:

1. Add an XL Carousel to a page with more than one image.
2. Save or Update page and view in browser.
3. Click on thumbnail images to influence which image is shown large.
4. Then try to tab to these thumbnail images - note that they cannot get keyboard focus.

### Expected behavior:

I would expect these thumbnail images to be contained in a button so that they can receive keyboard focus. Then when the Enter key or Space bar is depressed the javascript can change the large image. Screen reader users may need some extra labeling to explain what the buttons are for.

### Screenshots:

![image](https://user-images.githubusercontent.com/2565505/156891470-aa865329-1feb-4208-a402-be78c97e735c.png)
This screenshot shows the code for one of the thumbnail images. There is nothing here to get keyboard focus.

### Isolating the problem:

- [x] This bug happens with no other plugins activated
- [x] This bug happens with a default WordPress theme active
- [x] This bug happens **without** the Gutenberg plugin active
- [x] I can reproduce this bug consistently using the steps above

### WordPress version:

WP 5.9.1

### Gutenberg version:

Not using Gutenberg plugin.
